main.css

```diff
@@ -0,0 +1,227 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    }
+  
+  html{
+    font-size: 16px;
+    font-family: 'Poppins', sans-serif;
+  }
+  
+  body {
+      background-image: url("niebo.jpg");
+      background-size: cover;
+      color:#fff;
+  }
+  
+  .container {
+      display: flex;
+      justify-content: space-evenly;
+      align-items: center;
+      flex-direction: row;
+      flex-wrap: wrap;
+      max-width: 1200px;
+      margin: 0 auto;
+  }
+  .tile-styling{
+    display: flex;
+    width: 100%;  /*15rem dla pc*/
+    height: 8rem;
+    margin: 5px;
+    background-color: rgba(0, 0, 0, 0.5);
+    padding: 10px;
+    overflow: hidden;
+    overflow-wrap: break-word;
+  }
+  .general-data-tile{
+    display: flex;
+    width: 100%;  /*15rem dla pc*/
+    height: 8rem;
+    margin: 5px 10px;
+    background-color: rgba(0, 0, 0, 0.5);
+    padding: 10px;
+  }/*
+  .forecast-tile{
+  }
+  .details-tile{
+  }
+  .sunandmoon-tile{
+  }
+  .precipitation-tile{
+  }
+  .wind-pressure-tile{
+  }
+  .map-tile{
+  }
+  */
+  @media (min-width: 640px){
+    .tile-styling{
+      width: 48%;
+    }/*18.75rem;*/
+  }
+  @media (min-width: 768px){
+      .tile-styling{
+          width: 48%;
+      }/*23rem*/
+  }
+  @media (min-width: 1024px){
+    .tile-styling{
+      width: 48%;
+    }
+    .general-data-tile{
+      width: 100%;
+    }
+  }
+  
+  .navbar {
+      font-size: 12px;
+      background-image: linear-gradient(90deg,#001911 0%, #242aba 50%, #ffffff 100%);
+      border: 1px solid rgba(0, 0, 0, 0.2);
+      padding-bottom: 10px;
+      position: relative;
+      width: 100%;    
+  }
+  
+  .main-nav {
+      list-style-type: none;
+      display: none;
+  }
+  
+  .nav-links,
+  .logo {
+      text-decoration: none;
+      color: rgba(255, 255, 255, 1);
+  }
+  
+  .main-nav li {
+      text-align: center;
+      margin: 15px auto;
+  }
+  
+  .logo {
+      display: inline-block;
+      font-size: 22px;
+      margin: 15px 100px 10px 20px; 
```

Comment:
Używając flexa, możemy to idealnie wyśrodkować stosując align-items: center, wtedy nie potrzebujemy dodawać wartości dla margin top i bottom.